content/foundation/meeting-minutes/buying-bevy-org-domain/index.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Buying Bevy.org"
+date = 2025-06-02
+authors = ["Alice I. Cecile"]
++++
+
+<!-- more -->
+
+![Vote on buying Bevy.org domain](bevy-org-domain.png)
+
+**Proposal:** I have news on bevy.org:
+
+They will not sell it for $2,000, and initially said $6,000 is as low as they'll go,
+However they also made a very interesting offer: they would give it to us for free for a couple of years, with a contract stating we would pay $6,000 at the end of the contract, or transfer it back if we don't.,
+I said that was a interesting and generous offer / I'd run it by you all, but I suspected we would be hesitant to not have full ownership,
+I then said I have approval for $4,000 now, which he said was too low, but that he'd split the difference at $5,000.,
+
+So lets vote:
+
+Pay $5,000 for the domain,
+Take the domain for free now and enter a contract stating we'll pay $6,000 two years from now, or give it back,
+Do nothing / give up on the domain,
+
+My personal vote is (1). I think it is worth it, and while it isn't an inconsequential amount of money, it also isn't going to set us back meaningfully.
+
+**Proposed by:** Carter Anderson
```

Comment:
I think putting "proposed by" right after the Proposal (or combining it with that line), and then putting the proposal in "quote syntax" makes sense, in order to make who is speaking clear.